client/js/reducers/progress.js

```diff
@@ -11,21 +11,27 @@ const initialState = Immutable.fromJS({
   answerMessageIndex: '',
   responses: [],
   startedAt: 0,
-  finishedAt: 0
+  finishedAt: 0,
+  assessmentResult:null
 });
 
 export default (state = initialState, action) => {
 
   switch(action.type){
-    case AssessmentConstants.NEXT_ITEM:
+    case AssessmentConstants.ASSESSMENT_NEXT_QUESTION:
```

Comment:
It looks like all of these actions should be taken from assessment_progress, along with the assessment viewed.